avalon/tools/cbloader/lib.py

```diff
@@ -30,21 +30,29 @@ def refresh_family_config():
     # Update the icons from the project configuration
     project = io.find_one({"type": "project"},
                           projection={"config.families": True})
-    if not project:
-        raise RuntimeError("Project not found")
+
+    assert project, "Project not found!"
     families = project['config'].get("families", [])
     families = {family['name']: family for family in families}
 
+    # Check if any family state are being overwritten by the configuration
+    data = pipeline.get_data("family_states") or {}
+    family_states = {family: data.get(family, True) for family in families}
```

Comment:
How about removing this line.
And then on [L54](https://github.com/Colorbleed/core/pull/21/files#diff-aaea11967f14efbd277f8c0ffd0b8469R54) change it to:

```python
family["state"] = family_states.get(family)
```

```